packages/@coorpacademy-components/src/template/app-player/player/slides/style.css

```diff
@@ -105,16 +107,35 @@
 .guideWrapper {
   composes: infos;
   left: 0;
-  padding: 4px 10px;
-  border-bottom-right-radius: 12px;
   cursor: pointer;
-  border-top-left-radius: 3px;
+  border-radius: 3px;
+  margin-top: 24px;
+  margin-left: 24px;
+  padding: 12px;
 }
 
-.guideWrapper span {
+.guideWrapper[data-name="newMedia"] {
```

Comment:
pourquoi utiliser un data-name dans le css?
utilise une class CSS

Reply:
C'est noté ! je change ça. 
J'ai utilisé le data-name car il était déjà présent dans la structure et il me semble que ça me permet de cibler plus précisément, surtout que je n'ai pas l'impression que le CSS développé servira ailleurs. d'où ce choix de data-name plutôt que class.